src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_utils.py

```diff
@@ -49,49 +50,104 @@ def get_resource_group_name_by_resource_id(resource_id):
         resource_group_keyword): resource_id.index('/providers/')]
 
 
-def get_resource_group_name_by_registry_name(registry_name):
+def get_resource_group_name_by_registry_name(registry_name,
+                                             resource_group_name=None):
     """Returns the resource group name for the container registry.
     :param str registry_name: The name of container registry
+    :param str resource_group_name: The name of resource group
     """
-    arm_resource = _arm_get_resource_by_name(registry_name, ACR_RESOURCE_TYPE)
-    return get_resource_group_name_by_resource_id(arm_resource.id)
+    if resource_group_name is None:
```

Comment:
`if not resource_group_name`

Unless an empty group name is legal, which I don't think should be the case.